src/org/osflash/signals/Promise.as

```diff
@@ -50,7 +53,10 @@ package org.osflash.signals
         {
             if (_isDispatched)
             {
-                throw new IllegalOperationError("You cannot dispatch() a Promise more than once");
+				if (!ignoreSubsequentDipatches)
```

Comment:
Why not just use a OnceSignal?

Reply:
I wasn't aware I was submitting this as a pull request! I'm using promises for their retroactive dispatch functionality, and was sometimes getting runtime errors as promises sometimes got dispatched more than once. This just got around that issue.

Reply:
@esidegallery :dancer: